File: bos_web/src/main/webapp/js/service/userService.js
//服务层
app.service('userService',function($http){

    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../newCar/adminGetAllNewCar');
    }

    //搜索
    this.search=function(pageNo,pageSize){
        return $http.get('../user/findAll?pageNo=' + pageNo + "&pageSize=" + pageSize);
    }

});

File: bos_service/src/main/java/com/bos/service/UserService.java
package com.bos.service;


import com.bos.pojo.User;
import com.github.pagehelper.PageInfo;

import javax.servlet.http.HttpSession;

public interface UserService {
    /**
     * 查询所有用户
     * @return
     */
    public PageInfo<User> findAll(Integer pageNo, Integer pageSize, HttpSession session);

    /**
     * 添加用户信息
     * @param user
     * @param code
     * @param session
     * @return
     */
    public int addUser(User user, String code, HttpSession session);

    /**
     * 根据用户名和密码查询用户信息
     * @param user
     * @return
     */
    public User findOne(User user);

    /**
     * 修改用户信息
     * @param user
     */
    public int editUser(User user);
}
